### DISCUSSION TITLE
1.14.0 beta for 1.17

### DISCUSSION THREAD

**Discussion by Draylar** [Draylar/identity#228] [General]
1.17 update~ has only been minimally tested, so there is a good chance things will break. Please report any bugs you encounter!

<hr /><em>This discussion was created from the release <a href='https://github.com/Draylar/identity/releases/tag/1.14.0-beta-1.17'>1.14.0 beta for 1.17</a>.</em>

**Comment by Ch1216** [comment]
Are there gonna be custom abilities for the new mobs?

**Reply by James103** [reply]
For example, as a Goat, you should be able to ram towards an entity or block. As an Axolotl, if you take damage while at or below half health, with a certain probability (i.e. not always) mobs should not target you for a couple of seconds and you should receive Regeneration I for a few seconds.

**Reply by minecraftedleen** [reply]
also axololts should be hostile to fish, squid, drowned, and guardian identities
